### DISCUSSION TITLE
Will the vscode extension get CHANGELOG updates

### DISCUSSION THREAD

**Discussion by fisforfaheem** [community/community#125916] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

As a developer who relies on extensions to enhance my coding experience, I believe that including changelogs for all updates would be a significant improvement.

Changelogs provide users with crucial information about the changes and improvements made in each update. They allow us to stay informed about new features, bug fixes, and potential compatibility issues. This transparency is especially important for developers who rely on specific extensions for their daily workflow.

In addition to enhancing the overall user experience, changelogs also play a vital role in accessibility. By providing a clear and concise overview of the changes, users with disabilities can quickly determine whether an update addresses their specific needs or introduces any potential barriers.

I kindly request that you consider adding changelogs to all updates in the VS Code extensions tab. This simple addition would greatly benefit the entire developer community, including those with disabilities.

**Comment by cehfisher** [comment]
Thanks for posting in the GitHub Community, @fisforfaheem! We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category. The Accessibility category is a place for our community to discuss and provide feedback on the digital accessibility of GitHub products. Digital accessibility means that GitHub tools, and technologies, are designed and developed so that people with disabilities can use them. I've asked @queenofcorgis to move this issue into the correct category for you. Thank you!

**Comment by fisforfaheem** [comment]
Thanks
